graph/api/src/main/java/org/jboss/windup/graph/service/GraphService.java

```diff
@@ -298,14 +299,20 @@ public TitanTransaction newTransaction()
      *
      * @see GraphTypeManagerTest
      */
-    public static <T extends WindupVertexFrame> T addTypeToModel(GraphContext graphContext, WindupVertexFrame frame,
-                Class<T> type)
+    public static <T extends WindupVertexFrame> T addTypeToModel(GraphContext graphContext, WindupVertexFrame frame, Class<T> type)
     {
         Vertex vertex = frame.asVertex();
         graphContext.getGraphTypeRegistry().addTypeToElement(type, vertex);
         return graphContext.getFramed().frame(vertex, type);
     }
 
+    public T reload(T frame)
```

Comment:
This is missing javadoc, and I'm not sure that I like the name. "reload" implies that it is reloading the Vertex, but it is actually just reframing the already loaded vertex. Thoughts?